### PR TITLE
DZ8

### PR DESCRIPTION
**Pull request by Olga8878**

### CODE REVIEW

File: src/Lesson8/Phone.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+    }
+
+    public Phone(String number, String model) {
+        this.number = number;
+        this.model = model;
+
+    }
+
+    public Phone()
```
Comment (DHrytsuk) [inline src/Lesson8/Phone.java]: Usually, empty constructor placed first

File: src/Lesson8/Phone.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
+package Lesson8;
+
+
+public class Phone {
+    private String number;
+    private String model;
+    private String weight;
```
Comment (DHrytsuk) [inline src/Lesson8/Phone.java]: Why String?

File: src/Dop8/Patient.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+public class Patient {
+
+    private TreatmentPlan plan;
+    private Doctor doctor;
+
+    public Patient(TreatmentPlan plan) {
+        this.plan = plan;
+    }
+
+    public void assignDoctor() {
```
Comment (DHrytsuk) [inline src/Dop8/Patient.java]: Move it to Main with TreatmentPlan as a parameter

Comment (Olga8878) [reply src/Dop8/Patient.java]: Не очень понимаю, что именно перенести в main?

File: src/Lesson8/Phone.java (hunk 2 above)
Comment (DHrytsuk) [reply src/Lesson8/Phone.java]: What about weight?

File: src/Dop8/Patient.java (hunk 3 above)
Comment (DHrytsuk) [reply src/Dop8/Patient.java]: Method `assignDoctor`

File: src/Dop8/Main.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
+package Dop8;
+
+public class Main {
+    public static void main(String[] args) {
+
+        TreatmentPlan plan1 = new TreatmentPlan(2);
+        assignDoctor(plan1);
+    }
+
+    public static void assignDoctor(TreatmentPlan plan) {
```
Comment (DHrytsuk) [inline src/Dop8/Main.java:11]: We should set the doctor to the Patient based on its plan

**Review by DHrytsuk — APPROVED** [review]

**Pull request merged**